vars/elifeSpectrum.groovy

```diff
@@ -73,6 +73,7 @@ def call(Map parameters) {
                     def testLogArtifact = "${env.BUILD_TAG}.${environmentName}.log"
                     sh "cp ${env.SPECTRUM_PREFIX}build/test.log ${testLogArtifact}"
                     archive testLogArtifact
+                    sh "rm ${env.SPECTRUM_PREFIX}build/test.log"
```

Comment:
file is preserved as `testLogArtifact` on the filesystem too, so there should be no problem in removing. Don't need `-f` if it got here

Reply:
`junit.xml` can never be appended to like a log, so it can be removed immediately too

Reply:
Checked other custom steps, but they work on a Jenkins workspace folder that should be created afresh on each build. Will complete this step and merge.